doc/subsystems/logging/logger.rst

```diff
@@ -105,17 +105,18 @@ compiled in.
 :option:`CONFIG_LOG_PRINTK_MAX_STRING_LENGTH`: Maximal string length that can
 be processed by printk. Longer strings are trimmed.
 
+:option:`CONFIG_LOG_PROCESS_IN_IDLE`: When enabled, logger is processed in idle
+thread.
```

Comment:
Could we have a one option with simple choice (process log in idle, dedicated thread or user thread) instead of separated options?